fusion-plugin-rpc-redux-react/src/hook.js

```diff
@@ -0,0 +1,32 @@
+/** Copyright (c) 2018 Uber Technologies, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import React from 'react';
+import {FusionContext, useService} from 'fusion-react';
+import {RPCToken} from 'fusion-plugin-rpc';
+import {ReduxToken} from 'fusion-plugin-react-redux';
+import {createRPCHandler} from 'fusion-rpc-redux';
+
+type OptionalHandlerArgs = {
+  actions?: any,
+  transformParams?: (params: any) => any,
+  mapStateToParams?: (state: any, args?: any, ownProps: {}) => any,
+};
+
+export function useRPCHandler(rpcId: string, opts: OptionalHandlerArgs) {
```

Comment:
I think we should potentially hold off on exposing a useRPCHandler hook for a few reasons.

1. hooks cannot be used inside `prepared` functions. We could potentially refactor prepared to add support for this somehow (and maybe you already have that planned), but in the current state this doesn't work. This means that hooks could not be used for server side data fetching

2. I think the final API for this hook will look different when integrated into react suspense, which is the long term goal. I'm working on a proposal for what this could look like.

Reply:
I'm interested to see what your ideas with Suspense are. I figured you might have a better idea of where to go with this idea. It was really just a helper, so I'll revert it.

I guess I thought hooks did work in async rendering. Did this PR (https://github.com/fusionjs/fusionjs/pull/227) not address this?

Reply:
That PR fixed hooks inside components during the prepare cycle, but not in the prepare function itself. I think the majority of this PR looks good, just want to hold off on the hook specifically.